### DISCUSSION TITLE
Keeping older macOS versions around a bit longer

### DISCUSSION THREAD

**Discussion by adamhill** [buresdv/Cork#411] [Ideas]
I just read the newsletter, and I hope you know about Perception from [pointfree](https://www.pointfree.co/)

It's an amazing port of `@Observable` all the way back to _**macOS 10.15**_ (!!!) and as always – excellent explanations of the port on their blog with clear and tested code - [perception-a-back-port-of-observable](https://www.pointfree.co/blog/posts/129-perception-a-back-port-of-observable)

Thanks for Cork, I love exploring and seeing what's in a Formula or just seeing all the new stuff out there with an easy to use UI.

**Comment by buresdv** [comment]
Hey, I'm sorry I took so long to respond… I have a lot of stuff going on right now.

I have read about Perception, and it's a great library. I'd definitely use it.

Another (and arguably an even more important part) of why I started considering moving to 14 is that a lot of the concurrency stuff is only for 14+. I wanted to add features like live folder monitoring and dynamic notifications, but they're a bit bugged on 13.

It's a really difficult topic all around.
